bitnami/external-dns/Chart.yaml

```
@@ -24,4 +24,4 @@ sources:
   - https://github.com/kubernetes-sigs/external-dns
   - https://github.com/bitnami/containers/tree/main/bitnami/external-dns
   - https://github.com/kubernetes-sigs/external-dns
-version: 6.15.0
+version: 6.15.1
```

Adding functionality should imply a minor bump ([Semver summary](https://semver.org/#summary))

```suggestion
version: 6.16.0
```

---

Ah, of course, fixed here: https://github.com/bitnami/charts/pull/15804/commits/c4d3d4518a93a1d699400ee51ca37a549f2c9061